gtsam/sfm/TranslationRecovery.h

```diff
@@ -16,14 +16,16 @@
  * @brief Recovering translations in an epipolar graph when rotations are given.
  */
 
+#include <map>
```

Comment:
Include only what you really need in the header, include others in .cpp only

Reply:
std::map is used in line 59 below, for `sameTranslationNodes_`.